test/integration/controllermanager/shoot/maintenance/maintenance_test.go

```diff
@@ -342,6 +342,10 @@ var _ = Describe("Shoot Maintenance controller tests", func() {
 				g.Expect(testClient.Get(ctx, client.ObjectKeyFromObject(shoot), shoot)).To(Succeed())
 				g.Expect(*shoot.Spec.Provider.Workers[0].Machine.Image).To(Equal(gardencorev1beta1.ShootMachineImage{Name: highestAMD64MachineImage.Name, Version: highestAMD64MachineImage.Version}))
 				g.Expect(*shoot.Spec.Provider.Workers[1].Machine.Image).To(Equal(gardencorev1beta1.ShootMachineImage{Name: highestARM64MachineImage.Name, Version: highestARM64MachineImage.Version}))
+				g.Expect(shoot.Status.LastMaintenance).NotTo(BeNil())
+				g.Expect(shoot.Status.LastMaintenance.Description).To(Equal(
+					"Machine image of worker-pool \"cpu-worker1\" upgraded from \"foo-image\" version \"0.0.1-beta\" to version \"1.1.1\". Reason: AutoUpdate of MachineImage configured" + ", " +
+						"Machine image of worker-pool \"cpu-worker2\" upgraded from \"foo-image\" version \"0.0.1-beta\" to version \"1.2.0\". Reason: AutoUpdate of MachineImage configured"))
```

Comment:
Why do you only assert the `Description` field? The test could become more accurate by also asserting some other fields of the `LastMaintenance` struct. Same below.